src/Magnum/Audio/PlayableGroup.h

```diff
@@ -0,0 +1,194 @@
+
+    Copyright © 2010, 2011, 2012, 2013, 2014, 2015
+              Vladimír Vondruš <mosra@centrum.cz>
+    Copyright © 2015 Jonathan Hale <squareys@googlemail.com>
+
+    Permission is hereby granted, free of charge, to any person obtaining a
+    copy of this software and associated documentation files (the "Software"),
+    to deal in the Software without restriction, including without limitation
+    the rights to use, copy, modify, merge, publish, distribute, sublicense,
+    and/or sell copies of the Software, and to permit persons to whom the
+    Software is furnished to do so, subject to the following conditions:
+
+    The above copyright notice and this permission notice shall be included
+    in all copies or substantial portions of the Software.
+
+    THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+    IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+    FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+    THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+    LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+    FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+    DEALINGS IN THE SOFTWARE.
+*/
+
+/** @file
+ * @brief Class @ref Magnum::Audio::PlayableGroup, typedef @ref Magnum::Audio::PlayableGroup2D, @ref Magnum::Audio::PlayableGroup3D
+ */
+
+#include <string>
+
+#include <Magnum/SceneGraph/SceneGraph.h>
+#include <Magnum/SceneGraph/FeatureGroup.h>
+#include <Magnum/SceneGraph/AbstractFeature.h>
+
+#include "Magnum/Audio/Audio.h"
+#include "Magnum/Audio/Source.h"
+#include "Magnum/Audio/visibility.h"
+
+namespace Magnum { namespace Audio {
+
+/**
+@brief PlayableGroup
+
+Manages @ref Audio::Playable features and provides means of setting gain or transformation of
+a group of Playables, aswell as the ability of playing, pausing, stopping or cleaning all sources of all
+Playables.
+
+## Usage
+
+@code
+Object3D object;
+Source source;
+PlayableGroup3D group;
+Playable3D playable{object, source, &group};
+
+// ...
+
+object.translate(offset);
+
+// ... and every frame, update the sources positions:
+group.setAllClean();
+@endcode
+
+For two dimensional scenes simply replace `3D` with `2D`.
+
+@see @ref Playable, @ref SceneGraph::FeatureGroup
+*/
+template<UnsignedInt dimensions> class PlayableGroup: public SceneGraph::FeatureGroup<dimensions, Playable<dimensions>, Float> {
+    friend Playable<dimensions>;
+
+    public:
+
+        /**
+         * @brief Constructor
+         */
+        explicit PlayableGroup(): SceneGraph::FeatureGroup<dimensions, Playable<dimensions>, Float>{} {}
+
+        /**
+         * @brief Play all sound sources in this group
+         * @return Reference to self (for method chaining)
+         */
+        PlayableGroup<dimensions>& playAll() {
+            for(UnsignedInt i = 0; i < this->size(); ++i)
+                (*this)[i].source().play();
```

Comment:
Using [Source::play(...)](http://mosra.cz/blog/magnum-doc/classMagnum_1_1Audio_1_1Source.html#ace53ea6c28c35326c3265e029d906d90) will ensure that all sources start playing at the same time. I agree that current API of that function is not ideal, will fix it later.

Similarly for `pause()`/`stop()`/`rewind()` below.

Maybe naming these just `play()` (without the `All` suffix) could be enough?